templates/blog/article.html

```diff
@@ -45,10 +46,14 @@
   </div>
   {% endif %}
   
-  <h1 name=title>{{ article.title or 'Insert your title here' }}</h1>
-  <div class=content name=content>
+  <div id=editorTitle>
+    <h1>{{ article.title or 'Insert your title here' }}</h1>
```

Comment:
we need [this line](https://github.com/neomad-team/neomad.org/pull/228/files#diff-b2eb8af5867e7d92dac4de3e62798caeL48) to make the title editable